### ISSUE TITLE
No way to bump from pre-release to final release

### ISSUE THREAD

**Issue by patrick-jones** [royw/Versio#5]
According to [PEP 440](https://www.python.org/dev/peps/pep-0440/#pre-releases), pre-release versions should be followed by release versions with the same number, minus the pre-release qualifier. This is generally the case in most version systems; a pre-release (1.0.0rc1) is followed by a release (1.0.0). But as pointed out in the `readme`, Versio stops bumping at the end of a sequence. 

I think a more useful behavior would be to roll from pre-release version to release version:

```
>>> str(v1)
'1.2.3rc1'
>>> v1.bump('pre', 0)
True
>>> str(v1)
'1.2.3'
```

**Comment by royw** [comment]
Sorry I just now noticed your message. Sounds like a good correction.

**Comment by royw** [comment]
Added a promote attribute to bump which will remove the field after the last sequence:

    >>> v1 = Version('1.2.3rc1', scheme=Pep440VersionScheme)
    >>> str(v1)
    '1.2.3rc1'
    >>> v1.bump('pre', 0)
    False
    >>> str(v1)
    '1.2.3rc1'
    >>> v1.bump('pre', 0, promote=True)
    True
    >>> str(v1)
    '1.2.3'

Released as version 0.4.0 to pypi
